service/awsconfig/v7/version_bundle.go

```diff
@@ -49,7 +54,7 @@ func VersionBundle() versionbundle.Bundle {
 		Deprecated:   false,
 		Name:         "aws-operator",
 		Time:         time.Date(2018, time.February, 27, 14, 16, 0, 0, time.UTC),
-		Version:      "2.1.3",
+		Version:      "3.0.0",
```

Comment:
This is a backward incompatible change. We can not upgrade existing clusters.

Reply:
Ack. Major version bump makes sense in this case.